cmd/cluster-agent/app/compliance.go

```diff
@@ -90,6 +90,7 @@ func startCompliance(stopper restart.Stopper, apiCl *apiserver.APIClient, isLead
 	checkInterval := coreconfig.Datadog.GetDuration("compliance_config.check_interval")
 	checkMaxEvents := coreconfig.Datadog.GetInt("compliance_config.check_max_events_per_run")
 	configDir := coreconfig.Datadog.GetString("compliance_config.dir")
+	isRegoEvaluatorEnabled := coreconfig.Datadog.GetBool("compliance_config.rego_evaluator_enabled")
```

Comment:
nit: I'd probably lean more towards `compliance_config.enable_rego_evaluator` but it's just a style preference